tutorials/shield_multiphysics/doc/content/user_workshop/index.md

```diff
@@ -0,0 +1,286 @@
+
+!---
+
+# Step 3: Boundary conditions id=step03b
+
+## (continued)
+
+!include user_workshop/problem/step03.md start=end-intro
+
+!---
+
+!include user_workshop/problem/step04.md end=end-intro
+
+!---
+
+!include user_workshop/systems_user/materials.md
+
+!---
+
+!include user_workshop/systems_user/functions.md
+
+!---
+
+# Step 4: Heat Conduction with Material id=step04b
+
+## (continued)
+
+!include user_workshop/problem/step04.md start=end-intro
+
+!---
+
+!include user_workshop/problem/step05.md end=end-intro
+
+!---
+
+!include user_workshop/systems_user/auxkernels.md
+
+!---
+
+# Step 5: Heat Flux Auxiliary Variable id=step05b
+
+## (continued)
+
+!include user_workshop/problem/step05.md start=end-intro
+
+!---
+
+!include user_workshop/problem/step06.md end=end-intro
+
+!---
+
+!include user_workshop/systems_user/executioners.md
+
+!---
+
+!include user_workshop/systems_user/timeintegrators.md
+
+!---
+
+!include user_workshop/systems_user/timesteppers.md
+
+!---
+
+# Step 6: Transient Heat Conduction id=step06b
+
+## (continued)
+
+!include user_workshop/problem/step06.md start=end-intro
+
+!---
+
+!include user_workshop/problem/step07.md end=end-intro
+
+!---
+
+!include workshop/modules/modules.md
+
+!---
+
+# Step 7: Mechanics id=step07b
+
+## (continued)
+
+!include user_workshop/problem/step07.md start=end-intro
+
+!---
+
+!include user_workshop/problem/step08.md end=end-intro
+
+!---
+
+!include user_workshop/systems_user/adaptivity.md end=end-intro
+
+!---
+
+# Step 8: Mesh Adaptivity id=step08b
+
+## (continued)
+
+!include user_workshop/problem/step08.md start=end-intro
+
+!---
+
+!include user_workshop/problem/step09.md end=end-intro
+
+All MOOSE Postprocessors are based on the UserObject System, so we will begin with an introduction there.
+
+!---
+
+!include user_workshop/systems_user/userobjects.md
+
+!---
+
+!include user_workshop/systems_user/postprocessors.md
+
+!---
+
+!include user_workshop/systems_user/vectorpostprocessors.md
+
+!---
+
+# Step 9: Postprocessors
+
+## (continued)
+
+!include user_workshop/problem/step09.md start=end-intro
+
+!---
+
+!include user_workshop/problem/step10.md end=end-intro
+
+!---
+
+!include workshop/numerical/fvm_overview.md
+
+!---
+
+# Step 10: Finite Volume
+
+## (continued)
+
+!include user_workshop/problem/step10.md start=end-intro
+
+!---
+
+!include workshop/infrastructure/troubleshooting.md
+
+!---
+
+!include user_workshop/problem/step11.md end=end-intro
+
+!---
+
+!include user_workshop/systems_user/multiapps.md
+
+!---
+
+!include user_workshop/systems_user/transfers.md
+
+!---
+
+# Step 11: Multiscale Simulation id=step11b
+
+## (continued)
+
+!include user_workshop/problem/step11.md start=end-intro
+
+!---
+
+!include user_workshop/problem/step12.md end=end-intro
+
+!---
+
+!include user_workshop/systems_user/actions.md
+
+!---
+
+# Step 12: Custom Syntax id=step12b
+
+## (continued)
+
+!include user_workshop/problem/step12.md start=end-intro
```

Comment:
This step is mixing the custom syntax and multi-system. Could be split into 2 steps.

Reply:
This is how I intend Physics to be used and the case is not big enough to show that fully coupled is bad